spec/classes/mod/security_spec.rb

```diff
@@ -24,13 +24,20 @@
             )
           }
           it { is_expected.to contain_package('mod_security_crs') }
-          if facts[:os]['release']['major'].to_i > 6
+          if facts[:os]['release']['major'].to_i > 6 && facts[:os]['release']['major'].to_i <= 7
```

Comment:
maybe just `facts[:os]['release']['major'].to_i == 7` ?

Reply:
yeah we need to test 7.2 also hence added this condition(It failed on couple of OSes since release major version was 7.2)